sparkmagic/sparkmagic/livyclientlib/linearretrypolicy.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, seconds_to_sleep, max_retries):
         self._max_retries = max_retries
 
     def should_retry(self, status_code, error, retry_count):
-        return (status_code >= 500 and retry_count <= self._max_retries) or error
+        return (status_code is not None and status_code >= 500 and retry_count <= self._max_retries) or error
```

Comment:
This is now covered here: https://github.com/jupyter-incubator/sparkmagic/blob/master/sparkmagic/sparkmagic/livyclientlib/linearretrypolicy.py#L14

So please merge and remove this change.